runtime/moonbase/src/weights/cumulus_pallet_parachain_system.rs

```diff
@@ -61,10 +61,10 @@ impl<T: frame_system::Config> cumulus_pallet_parachain_system::WeightInfo for We
 		// Proof Size summary in bytes:
 		//  Measured:  `82`
 		//  Estimated: `3517`
-		// Minimum execution time: 2_128_000 picoseconds.
-		Weight::from_parts(91_853_295, 3517)
-			// Standard Error: 63_404
-			.saturating_add(Weight::from_parts(293_145_862, 0).saturating_mul(n.into()))
+		// Minimum execution time: 2_153_000 picoseconds.
```

Comment:
When I last run the benches for this pallet, the huge difference was corrected. It might be the machine entered in a peak of cpu, I don't know why. Now it makes sense (cc @RomarQ )